libs/partners/voyageai/langchain_voyageai/embeddings.py

```diff
@@ -55,7 +55,15 @@ def default_values(cls, values: dict) -> Any:
         model = values.get("model")
         batch_size = values.get("batch_size")
         if batch_size is None:
-            values["batch_size"] = 72 if model in ["voyage-2", "voyage-02"] else 7
+            values["batch_size"] = (
+                72
+                if model in ["voyage-2", "voyage-02"]
+                else (
+                    30
+                    if model == "voyage-3-lite"
+                    else (10 if model == "voyage-3" else 7)
+                )
+            )
```

Comment:
any chance this is set in the sdk as well? if so can we just make this `None` and rely on the sdk for defaults?

Reply:
@efriis Our SDK is thin and it is just a wrapper around the API. We let our clients to do the batching and do the batching in the integrations - wherever it is supported.